README.md

```diff
@@ -237,8 +238,8 @@ For a recommended workaround, see the code snippet below:
 
     // BETTER! Guarantees that the latest reference to fade is used
     useEffect(() => {
-        fade(0,1,5000)
-    }, [fade])
+        player && fade(0,1,5000)
```

Comment:
the initial vol makes sense, however before we merge can you try updating the version i released last night? I updated the implementation of methods like fade so that clients shouldn't have to depend on player for calls like this

Reply:
Right you are. I updated to 1.2.5 and it works without the player dependency.